### ISSUE TITLE
Additional Functions

### ISSUE THREAD

**Issue by lorenzschmid** [lorenzschmid/vlc-lamp#6]
If we just do lamps which communicate over VLC, no one will really care. Which is fine with me, since I do care :) But in order to impress other people (if that is a goal), I guess most will come down to the look and functions - which are mainly implemented in design and software. Again, I don’t expect us to do something revolutionary here, the market got it quite well cover. Some ideas:
* Waking up with light
* Adapting light to current...
 * music
 * weather
 * temperature
 * time
* Dynamic lightning (brightness sensor)
* ...

**Comment by lorenzschmid** [comment]
Raffael
 * Biorythm, mood or weather matching, ...
 * Room occupancy detecting
 * Gestures (no-touch) would be fancy to command the light-ambience

**Comment by sebastienbaumann** [comment]
Yeah I agree here, most solution are software based so we could add these functionalities on the way when we will have something working. For now we have to focus on the hardware needed:
- brightness sensor (simple variable resistor here would be enough I guess)
- motion detector. Any of you know a sensor which could be not too expensive?

**Comment by lorenzschmid** [comment]
Instead of a complete motion detector we could just use a couple of infrared diodes, no?
